src/content-fusion/content-fusion.ts

```diff
@@ -0,0 +1,24 @@
+import { apiClient } from '../erudio';
+import { StructureNodeOutputDto } from '../content-fusion/dto/structure-node-output.dto';
+
+export class ContentFusion {
+  private base_url: string;
+
+  constructor(base_url: string) {
+    this.base_url = base_url;
+  }
+
+  public async getNode(structure_id: string, locale: string): Promise<any> {
+    const url = this.base_url + `/content/${structure_id}/`;
```

Comment:
@JERILJOSE96 use one way of concatenating strings; template strings (with backticks ```) is preferred.